packages/yarnpkg-pnp/sources/loader/applyPatch.ts

```diff
@@ -316,6 +316,7 @@ export function applyPatch(pnpapi: PnpApi, opts: ApplyPatchOptions) {
     if (requireStack.length > 0)
       firstError.message += `\nRequire stack:\n- ${requireStack.join(`\n- `)}`;
 
+    Error.captureStackTrace(firstError);
```

Comment:
Should probably only do this if the error has a `pnpCode`, having it hide the stack of other errors would be a pain when working on the pnpapi